frontends/PyCDE/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+cocotb>=1.6.2
```

Comment:
I assumed that I should make this as only the PyCDE requirements from the toml. Let me know if that assumption is incorrect.